### DISCUSSION TITLE
In Brave browser, it‘s option is hidden

### DISCUSSION THREAD

**Discussion by TimDaub** [rainbow-me/rainbowkit#1405] [Feedback]
I think expecting the user here to horizontal scroll (especially without affordance) basically kills all engagement to a point where only expert users will know what to do

https://github.com/rainbow-me/rainbowkit/assets/2758453/2c25cfb9-1cc6-41f4-8164-8b85d5286b11

**Comment by DanielSinclair** [comment]
What wallet option are you missing/hoping to see?

**Reply by TimDaub** [reply]
brave (IMO the obvious option in brave) is hidden and the user has to scroll to the right to reveal it. But there is no affordance that would suggest that horizontal scrolling is possible

**Reply by DanielSinclair** [reply]
I see. This is something we'll look at. On desktop, we already pin the current injected wallet to the first position, so we can recreate that behavior on mobile. We also do plan to redesign and improve the wallet scrolling here.
